pkg/apis/events/event-data.go

```diff
@@ -101,6 +101,29 @@ type AzureEventsHubEventData struct {
 	Metadata map[string]string `json:"metadata,omitempty"`
 }
 
+type AzureServiceBusEventData struct {
+	// ApplicationProperties can be used to store custom metadata for a message
+	ApplicationProperties map[string]interface{} `json:"applicationProperties"`
+	// Message body
+	Body interface{} `json:"body"`
+	// ContentType is the MIME content type
+	ContentType *string `json:"contentType"`
+	// CorrelationId is the correlation identifier
+	CorrelationId *string `json:"correlationId"`
```

Comment:
Use `CorrelationID *string` and `json:"correlationID"`, similar to 

https://github.com/kubernetes/kubernetes/blob/3ffdfbe286ebcea5d75617da6accaf67f815e0cf/staging/src/k8s.io/api/core/v1/types.go#L879

Reply:
Changed it to be `CorrelationID`